### PR TITLE
l10n: up-to-date Russian "rules"

### PR DESCRIPTION
**Pull request by ihptru**

### CODE REVIEW

**Comment by ihptru** [comment]
http://i.imgur.com/vEBeS.png

File: mods/ra/chrome/ru/ingame.yaml [hunk 1]
```diff
@@ -63,67 +72,66 @@ Container@INGAME_ROOT:
...
 					Logic:OrderButtonsChromeLogic
 					X:3
 					Y:0
 					Width:30
 					Height:30
 					Image:sell
-					Description:Продать
-					LongDesc:Продавать строения, за часть \nих стоимости
+					Description:Sell
+					LongDesc:Sell buildings, reclaiming a \nproportion of their build cost
```
Comment (Mailaender) [inline mods/ra/chrome/ru/ingame.yaml]: I guess you did this by accident.

File: mods/ra/chrome/ru/ingame.yaml [hunk 2]
```diff
@@ -36,25 +31,39 @@ Container@INGAME_ROOT:
...
+			HoldText: ON HOLD
 		BuildPalette@INGAME_BUILD_PALETTE:
-			Id:INGAME_BUILD_PALETTE
 			X:WINDOW_RIGHT - 250
 			Y:280
 			Width:250
 			Height:500
+			ReadyText: READY
+			HoldText: ON HOLD
+			RequiresText: Requires
```
Comment (Mailaender) [inline mods/ra/chrome/ru/ingame.yaml]: You can also translate those.

Comment (ihptru) [inline mods/ra/chrome/ru/ingame.yaml]: missed

File: mods/ra/chrome/ru/ingame.yaml (hunk 1 above)
Comment (ihptru) [inline mods/ra/chrome/ru/ingame.yaml]: I forgot to translate this
I did not use old translation, I translated chrome from zero

**Comment by Mailaender** [comment]
Thanks

**Pull request merged**